### ISSUE TITLE
Set State example runs in SimpleIDE, but not Blockly (WAIT function)

### ISSUE THREAD

**Issue by kgracey** [parallaxinc/BlocklyProp#238]
I created the following blocks in Blockly, connected to a seven-segment LED. Please note the function name is "wait":
![screen shot 2016-06-13 at 2 42 45 pm](https://cloud.githubusercontent.com/assets/9534148/16023866/a032f578-3173-11e6-9888-74a1d792a86f.png)
It produces the following code from Blockly:
```
#include "simpletools.h"

void display0() {
  set_outputs(15, 8, 0b11100111);
}

void display1() {
  set_outputs(15, 8, 0b00100001);
}

void wait() {
  pause(500);
}


int main() {

  for (int n = 0; n < 1; n++) {
    set_directions(15, 8, 0b11111111);
    display0();
    wait();
    display1();
    wait();
  }

}
```
And it throws this error during RAM/EEPROM download:
Required libraries: simpletools
Library compile order: None needed
single.c: In function 'wait':
single.c:11:1: error: number of arguments doesn't match prototype
/opt/parallax/Workspace/Learn/Simple Libraries/Utility/libsimpletools/simpletools.h:1531:6: error: prototype declaration
single.c: In function 'main':
single.c:21:5: error: too few arguments to function 'wait'
single.c:11:6: note: declared here
single.c:23:5: error: too few arguments to function 'wait'
single.c:11:6: note: declared here

Now, here's the gotcha. This code can be copied into SimpleIDE and it compiles and runs, but not in Blockly. The WAIT function exists in SimpleIDE, but the WAIT function has been deprecated. Apparently Simpletools.h has the WAIT function, but it doesn't get angry that we're trying to use WAIT. However, Blockly should choke on WAIT (and so should SImpleIDE). 

**Change the function name to anything else (delay?). Now it compiles in Blockly.** 

Where does this problem lie? I'm using SimpleIDE 1.02 which doesn't show the error, but the latest internal version of SimpleIDE does show the error. 

Is it possible to create a list of words that shouldn't be used as function names? Is that the best way to handle this in Blockly? It seems that this is ultimately the error logs created in SimpleIDE need to be repackaged in plain English. 

@PropGit what are your thoughts? Is there a way in Blockly to handle this issue? Can a list of keywords be compiled and kept in Blockly so they don't get misused in this way, generating the cryptic error log?

Ken Gracey

**Comment by michel-cf** [comment]
I added `wait` to the list of reserved words.

![image](https://cloud.githubusercontent.com/assets/5745991/16060003/e5d592ee-3285-11e6-8a45-33a40dbf64ef.png)

![image](https://cloud.githubusercontent.com/assets/5745991/16060016/f60ed26a-3285-11e6-92ed-854580475eda.png)
